src/frontend/src/styles/main.css

```diff
@@ -2295,6 +2295,11 @@ a.c-action-list__item {
   text-align: center;
 }
 
+.c-progress-stepper--narrow {
+  padding-left: 10%;
```

Comment:
Just to be sure, you on purpose used percent right?

all other paddings of the dapp seems to be expressed in relative size `em`.

Reply:
@peterpeterparker yes, especially on mobile I want the space to not be a fixed size, in order to alway look appropriate! Thanks for cheking!